dolphinscheduler-ui/src/locales/zh_CN/project.ts

```diff
@@ -644,6 +644,12 @@ export default {
     zeppelin_parameters_tips: '请输入zeppelin dynamic form参数',
     zeppelin_rest_endpoint: 'zeppelinRestEndpoint',
     zeppelin_rest_endpoint_tips: '请输入zeppelin server的rest endpoint',
+    hive_cli_task_execution_type: 'Hive Cli 任务类型',
+    hive_cli_task_execution_type_tips: '请选择您的Hive Cli任务类型',
```

Comment:
This seems not to be used, same as the en_US.

Reply:
Sure will remove it